skyvern/forge/api_app.py

```diff
@@ -129,3 +134,38 @@ async def request_middleware(request: Request, call_next: Callable[[Request], Aw
 
 
 app = get_agent_app()
+
+# Register MCP server
+mcp_server = add_mcp_server(
+    app,
+    mount_path="/mcp",
+    name="Skyvern-MCP",
+    description="MCP server for Skyvern",
+    describe_all_responses=False,  # False by default. Include all possible response schemas in tool descriptions, instead of just the successful response.
+    describe_full_response_schema=False,  # False by default. Include full JSON schema in tool descriptions, instead of just an LLM-friendly response example.
+)
+
+
+async def _skyvern_run_task_v1(user_prompt: str, url: str) -> Any | None:
+    skyvern_agent = SkyvernAgent()
+    llm_prompt = prompt_engine.load_prompt("generate-task", user_prompt=user_prompt)
+    llm_response = await app.LLM_API_HANDLER(prompt=llm_prompt, prompt_name="generate-task")
+    task_generation = TaskGenerationBase.model_validate(llm_response)
+    task_request = TaskRequest.model_validate(task_generation, from_attributes=True)
+    if url is not None:
+        task_request.url = url
+    return await skyvern_agent.run_task(prompt=user_prompt, url=url)
+
+
+@mcp_server.tool()
+async def skyvern_v1(user_goal: str, url: str) -> dict:
```

Comment:
let's just name it skyvern_task

```suggestion
async def skyvern_v1(prompt: str, url: str) -> dict:
```

Reply:
We've fully launched the new interface. Here are example code to call the agent locally or call the remotely:
```
# run locally
from skyvern import SkyvernAgent
agent = SkyvernAgent()
await agent.run_task(prompt="get the top post on hackernews")

# call skyvern api
from skyvern import SkyvernAgent
agent = SkyvernAgent(base_url="https://api.skyvern.com", api_key="API KEY")
await agent.run_task(prompt="get the top post on hackernews")

```